bezier-rs/lib/src/lib.rs

```diff
@@ -305,4 +305,58 @@ impl Bezier {
 		};
 		bezier_starting_at_t1.split(adjusted_t2)[t2_split_side]
 	}
+
+	/// Returns two lists of t-values representing the local extrema of the x and y parametric curves respectively
+	/// The local extrema are defined to be points at which the derivative of the curve is equal to zero
+	fn _local_extrema(&self) -> [Vec<f64>; 2] {
+		let mut x_values = Vec::new();
+		let mut y_values = Vec::new();
+		match self.handles {
+			BezierHandles::Quadratic { handle } => {
+				let a = handle - self.start;
+				let b = self.end - handle;
+				// check for x extrema
+				if (b.x - a.x) != 0. {
+					x_values.push(-a.x / (b.x - a.x));
```

Comment:
Minor comment: It probably won't make that much of a difference, but you can avoid recomputing the `b.x - a.x` by storing the difference earlier.